qna/FR-add-cashbox-local-fr.md

```diff
@@ -1,31 +1,119 @@
 ## Question
-Comment créer une CashBox (local) ?
+
+Comment créer le produit ChaîneLocale ?
 
 ## Metadata tags
-lang-fr, market-fr, middleware, PosCreator, PosDealer, PosOperator, Consultant
 
-## Answer
-Pour créer une CashBox, certaines conditions doivent être remplies au préalable : 
+lang-de, lang-at, market-fr, middleware, PosCreator, PosDealer, PosOperator, Consultant
+
+## Réponse
+
+![ChaîneLocale](../images/FR/product_ChaineLocale.png)
+
+Le produit ChaîneLocale permet l'utilisation du fiskaltrust.Middleware sur un POS-System avec une installation locale. Ce produit est gratuit et peut être créé à tout moment via le [fiskaltrust.Portal] (https://proral.fiskaltrust.fr/). Plusieurs conditions doivent être remplies préalablement.
+
+### Exigences
+
+Inscrivez-vous en tant que revendeur (PosDealer) sur le [fiskaltrust.Portal] (https://protal.fiskaltrust.fr/).
+
+#### Activer l'utilisateur de caisse (PosOperator)
+
+Ouvrez le menu _PosOperator_ et la page _Vue d'ensemble_ puis cliquez sur la loupe dans le premier widget.
+
+![Vue d'ensemble PosOperator](../images/FR/step_by_step_chainelocale_001.jpg)
+
+Cliquez sur le nom du PosOperator correspondant dans la première colonne pour activer le compte du PosOperator.
+
+![List de PosOperators](../images/FR/step_by_step_chainelocale_002.jpg)
+
+#### Entrer le SIREN et le nom de l'entreprise
+
+Tout d'abord, un SIREN valide doit être saisi et vérifié dans les données de base de l'entreprise. Pour ce faire, vous devez compléter les données de base de l'entreprise.
+
+![Vérifiez le nom de la société et le SIREN](../images/FR/step_by_step_chainelocale_003.jpg)
+
+Inscrivez la raison sociale telle qu'elle figure dans le registre du commerce dans le champs **1**. Dans le champs **2**, entrez le SIREN de l'entreprise. (En mode "Sandbox", n'importe quel numéro à 9 chiffres peut être utilisé, par exemple 987654321). Cliquez ensuite sur le bouton de contrôle **3** pour comparer le SIREN avec le registre du commerce. Enregistrez les données à l'aide du bouton situé en bas de la fenêtre.
+
+#### Ajouter des établissement et le SIRET
+
+Cliquez sur le sous-menu _Outlets_ à la fin du menu _Nom d'entreprise_ pour afficher la liste des établissement (points de vente).
+
+![Liste des agences](../images/FR/step_by_step_chainelocale_004.jpg)
```

Comment:
There were some wrong translations "sortie primaire",... I have updated it in https://translation.fiskaltrust.cloud/.
Screen shot should be redo